Automattic/wp-calypso
# Jetpack Cloud: Remove Backups Detail Page

#### Changes proposed in this Pull Request

* Remove Backups Detail Page

#### Testing instructions

1. Navigate through the backups section and make sure no functionality is broken other than the "Change in this backup" links
